README.md

```diff
@@ -31,5 +31,5 @@ PLEASE, MAKE A PULL REQUEST SO WE CAN FINISH THIS PROJECT ASAP. Also, give it a
 * [youtube/sammillervlogs](https://youtube.com/sammillervlogs)
 
 ### License
-Copyright © 2021, [Samuel Mariña](https://github.com/samuelmarina).
+Copyright © 2021-2022, [Samuel Mariña](https://github.com/samuelmarina).
```

Comment:
Change it to 2021-present 🤡

Reply:
Then we'd only be present-proofing it, not future-proofing it.

Reply:
if you present-proof it, it will be future proof in future as well.

Reply:
Maybe write `2021-future` here?

Reply:
Yeah, we have to talk with core team😂

Reply:
Why not use "to Infinity and beyond"? You guys really need to get your priorities together 🤣

Reply:
Shh! Don't use laughing emoji in a serious discussion, 🤫

Reply:
this lib is going to be used on blue origin's moons rover, its going to the moooon.

Reply:
Another idea to settle this heated debate, maybe 42?

Reply:
Could we make it timeless?

> Copyright © a long time ago (in this galaxy) - to infinity (and beyond)